src/NHibernate/Async/Engine/ActionQueue.cs

```diff
@@ -24,15 +24,6 @@ namespace NHibernate.Engine
 	using System.Threading;
 	public partial class ActionQueue
 	{
-
-		public Task AddActionAsync(BulkOperationCleanupAction cleanupAction, CancellationToken cancellationToken)
```

Comment:
Please manually add back this method (into non-async part) as an async-to-sync wrapper to AddAction. And make it obsolete.

Reply:
It happened automatically, but I'll try what you said.

Reply:
Unfortunately, making it obsolete didn't stop the async generator from generating calls to it, and since that is treated as an error, it prevents compilation. I fixed it manually, for the sake of the review, but is there a way to prevent this?